src/index.html

```diff
@@ -13,6 +13,55 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
```

Comment:
According to the checklist, it's recommended to use semantic tags where possible. Instead of using the 'div' tag, consider using the 'section' tag for each group of stars. This will improve your page SEO and helps screen readers.